src/CreatorToken.sol

```diff
@@ -124,10 +176,20 @@ contract CreatorToken is ERC721Royalty {
     payToken.safeTransferFrom(msg.sender, admin, _adminFee);
   }
 
+  /// @notice Buy multiple creator tokens in bulk and send them to the caller.
```

Comment:
nit: recommend "mint them to" rather than send them to.

Reply:
Changed in https://github.com/showtime-xyz/creator-tokens/commit/3c46721e1c9a0899f54630e2b73028cd5bef159c